packages/docs/docs/providers/saml-jackson.md

```diff
@@ -0,0 +1,58 @@
+---
+id: saml-jackson
+title: BoxyHQ SAML Jackson
+---
+
+## Documentation
+
+BoxyHQ SAML Jackson is an open source service that handles the SAML login flow as an OAuth 2.0 flow, abstracting away all the complexities of the SAML protocol.
+
+You can deploy SAML Jackson as a separate service or embed it into your app using our NPM library. [Please check our documentation for more details](https://boxyhq.com/docs/jackson/deploy)
+
+## Configuration
+
+SAML login requires a configuration for every tenant of yours. One common method is to use the domain for an email address to figure out which tenant they belong to. You can also use a unique tenant ID (string) from your backend for this, typically some kind of account or organization ID.
+
+Check our [documentation](https://boxyhq.com/docs/jackson/getting-started#2-saml-config-api) for more details.
+
+## Options
+
+The **SAML Jackson Provider** comes with a set of default options:
+
+- [BoxyHQ Provider options](https://github.com/nextauthjs/next-auth/blob/main/src/providers/boxyhq.ts)
```

Comment:
```suggestion
- [BoxyHQ Provider options](https://github.com/nextauthjs/next-auth/tree/main/packages/next-auth/src/providers/boxyhq.ts)
```

Let's keep the filename/id the same everywhere. You can decide on which one to use. `boxyhq` or `saml-jackson` or something else?